ale/drivers/dawn_driver.py

```diff
@@ -0,0 +1,126 @@
+        Returns latest instrument metakernels
+
+        Returns
+        -------
+        : string
+          Path to latest metakernel file
+        """
+        metakernel_dir = config.dawn
+        mks = sorted(glob(os.path.join(metakernel_dir,'*.tm')))
+        if not hasattr(self, '_metakernel'):
+            for mk in mks:
+                if str(self.start_time.year) in os.path.basename(mk):
+                    self._metakernel = mk
+        return self._metakernel
+        
+    @property
+    def instrument_id(self):
+        """
+        Returns an instrument id for uniquely identifying the instrument, but often
+        also used to be piped into Spice Kernels to acquire IKIDs. Therefore the
+        the same ID that Spice expects in bods2c calls.
+
+        Returns
+        -------
+        : str
+          instrument id
+        """
+        instrument_id = label["INSTRUMENT_ID"]
+        filter_number = label["FILTER_NUMBER"]
+        
+        return "DAWN_{}_FILTER_{}".format(instrument_id, filter_number)
+    
+    @property
+    def _odtk(self):
+        """
+        Returns
+        -------
+        : list
+          Radial distortion coefficients
+        """
+        return spice.gdpool('INS{}_RAD_DIST_COEFF'.format(self.ikid),0, 1).tolist()
+    
+    @property
+    def label(self):
+        """
+        Loads a PVL from from the _file attribute and
+        parses the binary table data.
+
+        Returns
+        -------
+        PVLModule :
+            Dict-like object with PVL keys
+        """
+        if not hasattr(self, "_label"):
+            if isinstance(self._file, pvl.PVLModule):
+                self._label = self._file
+            try:
+                self._file.replace("\\", "/")
+                self._label = pvl.loads(self._file)
+            except Exception:
+                slabel = open(self._file).read().replace("\\", "/")
+                self._label = pvl.loads(slabel)
+            except:
+                raise ValueError("{} is not a valid label".format(self._file))
+        return self._label
+    
+    @property
+    def spacecraft_name(self):
+        """
+        Spacecraft name used in various Spice calls to acquire
+        ephemeris data.
+
+        Returns
+        -------
+        : str
+          Spacecraft name
+        """
+        return 'DAWN'
+    
+    @property
+    def target_name(self):
+        """
+        Returns an target name for unquely identifying the instrument, but often
+        piped into Spice Kernels to acquire Ephermis data from Spice. Therefore they
+        the same ID the Spice expects in bodvrd calls.
+
+        Returns
+        -------
+        : str
+          target name
+        """
+        return "VESTA"
+    
+    @property
+    def center_ephemeris_time(self):
+        """
+        The center ephemeris time for a framer.
+        """
+        center_time = self.starting_ephemeris_time + self.exposure_duration / 2
+        return center_time
```

Comment:
This looks like the generic framer approach, so hopefully we don't need to override it in a specific driver?

Reply:
Apparently, the base Framer doesn't have it. So probably better to move it into the base framer class.

Reply:
It looks like by the way the inheritance is happening, the Spice call to this method would be used - which (for some reason) has the logic for a LS. Maybe this would take even more reworking of the base class?